src/test/java/uk/gov/companieshouse/items/orders/api/controller/CertificateItemsControllerUnitTest.java

```diff
@@ -31,48 +36,64 @@
     @Autowired
     private ObjectMapper objectMapper;
 
+    @Autowired
+    private CertificateItemRepository repository;
+
+    private static final String EXPECTED_ITEM_ID = "CHS00000000000000001";
+
     @Test
     @DisplayName("Create creates certificate item")
     void createCertificateItemCreatesCertificateItem() throws Exception {
 
         // Given
-        final CertificateItemDTO newCertificateItemDTO = new CertificateItemDTO();
-        newCertificateItemDTO.setCompanyNumber("1234");
+        final CertificateItemDTO newItem = new CertificateItemDTO();
+        newItem.setCompanyNumber("1234");
         final CertificateItemOptions options = new CertificateItemOptions();
         options.setCertInc(true);
         options.setCertShar(true);
-        newCertificateItemDTO.setItemOptions(options);
-        newCertificateItemDTO.setQuantity(5);
+        newItem.setItemOptions(options);
+        newItem.setQuantity(5);
```

Comment:
Can the numbers be refactored into constants please? This one is duplicated in the new and expected item.

Reply:
OK, will do.